src/metrics/metrics.js

```diff
@@ -120,5 +122,13 @@ export default class Metrics {
     });
 
     this.baggageTruncate = this._factory.createCounter('baggage-trucate');
+
+    this.throttlerUpdateSuccess = this._factory.createCounter('throttler-update', {
```

Comment:
I don't know the params for this factory, but should this string be something like `throttler-update-success` and the next one `throttler-update-failure`?